soknad/src/main/java/no/nav/k9/søknad/felles/aktivitet/ArbeidAktivitet.java

```diff
@@ -16,22 +16,26 @@
     @JsonInclude(value = JsonInclude.Include.NON_EMPTY)
     @JsonProperty(value = "selvstendigNæringsdrivende")
     @Valid
-    private final List<SelvstendigNæringsdrivende> selvstendigNæringsdrivende;
+    private List<SelvstendigNæringsdrivende> selvstendigNæringsdrivende;
 
     @JsonInclude(value = JsonInclude.Include.NON_EMPTY)
     @JsonProperty(value = "frilanser")
     @Valid
-    private final Frilanser frilanser;
+    private Frilanser frilanser;
```

Comment:
Trenger du ikke en get for selvstendigNæringsdrivende, frilanser? Burde også være imutable?

Reply:
det fins allerede?